extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -188,7 +225,20 @@ export class AmpA4A extends AMP.BaseElement {
     this.experimentalNonAmpCreativeRenderMethod_ =
       platformFor(this.win).isIos() ? XORIGIN_MODE.SAFEFRAME : null;
 
-    this.emitLifecycleEvent('adSlotBuilt');
+
+    /**
+     * Protected version of emitLifecycleEvent that ensures error does not
+     * cause promise chain to reject.
+     * @private {!function(string, !Object=)}
```

Comment:
[`function` (with a lowercase `f`) is considered a primitive type and is non-nullable by default; do not prefix it with `!`.](https://google.github.io/styleguide/jsguide.html#jsdoc-nullability)